dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/deploy/DefaultApplicationDeployer.java

```diff
@@ -358,9 +363,11 @@ private void initMetricsService() {
     }
 
     private void initMetricsReporter() {
-        DefaultMetricsCollector collector =
-            applicationModel.getFrameworkModel().getBeanFactory().getOrRegisterBean(DefaultMetricsCollector.class);
+        ScopeBeanFactory beanFactory = applicationModel.getFrameworkModel().getBeanFactory();
+        DefaultMetricsCollector collector = beanFactory.getOrRegisterBean(DefaultMetricsCollector.class);
         MetricsConfig metricsConfig = configManager.getMetrics().orElse(null);
+        beanFactory.registerBean(GlobalMetricsEventMulticaster.class);
+
```

Comment:
Move to `org.apache.dubbo.rpc.model.ScopeModelInitializer` like `org.apache.dubbo.config.ConfigScopeModelInitializer`, `org.apache.dubbo.registry.RegistryScopeModelInitializer` and `org.apache.dubbo.common.CommonScopeModelInitializer`.